testing/test_assertion.py

```diff
@@ -17,12 +17,23 @@
 from _pytest.pytester import Pytester
 
 
-def mock_config(verbose=0):
-    class Config:
-        def getoption(self, name):
-            if name == "verbose":
+def mock_config(verbose: int = 0, assertion_override: Optional[int] = None):
```

Comment:
With the move away from `getoption()`, this needed to get a bit more complicated. (In the long term, it would probably be beneficial move the code base to a consistent way to access the verbosity level.)